styles.css

```diff
@@ -352,4 +352,8 @@ td.selected {
 
 #page-mod-questionnaire-questions #region-main .mform .fitem .felement {
     margin-bottom: 0;
-}
\ No newline at end of file
+}
+
+#page-mod-questionnaire-view h1 {
```

Comment:
Reason for this style? Is it targeting something dynamic? Are you not able to use the bootstrap heading classes for this instead?

Reply:
the above is placed in a text input, not allowing for html markup, went this route so that i can return h1 tag instead of starting off with an h2 on the mustache file